### PR TITLE
Fix imaging extensions

### PR DESCRIPTION
**Pull request by adamjtaylor**
* Add `Imaging` class to replace original OmeTif class.

### CODE REVIEW

File: fileformats [hunk 1]
```diff
@@ -0,0 +1 @@
+1sc, 2fl, acff, afi, afm, aiix, aim, aisf, al3d, ali, am, amiramesh, ano, apl, arf, atsf, avi, bif, bin, bip, bmp, btf, c01, cfg, ch5, cif, companion.ome, cr2, crw, csv, cxd, czi, dat, db, dcm, df3, dib, dic, dicom, dm2, dm3, dm4, dti, dv, dv.log, env, eps, epsi, ets, exp, fake, fdf, fff, ffr, fits, flex, fli, frm, fts, gel, gif, grey, h5, hdf, hdr, hed, his, htd, htm, html, hx, i2i, ics, ids, im3, ima, img, ims, inf, inr, ipl, ipm, ips, ipw, j2k, j2ki, j2kr, jp2, jpe, jpeg, jpf, jpg, jpk, jpx, klb, l2d, labels, lei, lif, liff, lim, lms, log, lsm, lut, map, mdb, mea, mnc, mng, mod, mov, mrc, mrcs, mrw, msr, mtb, mvd2, naf, nd, nd2, ndpi, ndpis, nef, nhdr, nii, nii.gz, nrrd, obf, oib, oif, oir, ome, ome.btf, ome.tf2, ome.tf8, ome.tif, ome.tiff, ome.xml, omp2info, par, pbm, pcoraw, pct, pcx, pgm, pic, pict, png, pnl, ppm, pr3, ps, psd, pst, pty, qptiff, r3d, r3d_d3d, r3d.log, raw, rcpnl, rec, res, scan, scn, sdt, seq, set, sif, sld, sm2, sm3, spc, spe, spi, spl, st, stk, stp, svs, sxm, tf2, tf8, tfr, tga, thm, tif, tiff, tim, tnb, top, txt, v, vff, vms, vsi, vws, wat, wav, wlz, wpi, xdce, xlog, xml, xqd, xqf, xv, xys, zfp, zfr, zip, zpo, zvi
```
Comment (thomasyu888) [inline fileformats:1]: Thats a lot of file formats.  I also see `zip` in there.  I suggest that we start small and add as we go.

Comment (adamjtaylor) [reply fileformats:1]: Agreed... I'll filter by those we actually have seen.... Technically all of these can be used to store bioformats compatible images though and so in the future would be fair game. I'll manually curate for now

**Review by thomasyu888 — APPROVED** [review]
LGTM.  Another downside of this is that the validation status table won't have granular file types .  It will all display as "Imaging"

**Comment by adamjtaylor** [comment]
I think that’s OK for now. But I do see the limitation. Thanks Tom!

**Pull request merged**